config.lua

```diff
@@ -1,18 +1,18 @@
 Config = Config or {}
 
 Config.AllowedVehicles = {
-    [1] = {model = `bus`}
+    {model = `bus`}
```

Comment:
Seems like having the model as an explicit field is unneeded. The Config field could just be named Config.AllowedVehicleModels. Optional.

Reply:
I left it there because that whole table is sent to the event as the `args`, so if anyone wants to add on to it, it's easy